README.md

```diff
@@ -1 +1,30 @@
-# office-helper
\ No newline at end of file
+office-helper
+=============
+
+Requirements
+------------
+
+- maven 3
+- jdk 1.8
+- docker or mysql server
+
+
+How to install
+--------------
+
+Clone the repository and cd into it
+
+    git clone git@github.com:syouon/office-helper.git
+    cd office-helper
+
+build and start the mysql server container using docker
+
+    sh tools/docker/run.sh start
+
+build the project with maven
+
+    mvn package
+
+run the application
+
+    mvn spring-boot:run
```

Comment:
Si on suit les steps du readme, ça ne se lance pas ! Quand je tape `mvn spring-boot:run` la console me crache un
`Failed to execute goal org.springframework.boot:spring-boot-maven-plugin:1.5.1.RELEASE:run (default-cli) on project office-helper: Unable to find a suitable main class, please add a 'mainClass' property`
Je suppose que le plugin est pas configuré.